src/app/view/components/dot-language-selector/dot-language-selector.component.spec.ts

```diff
@@ -8,11 +8,14 @@ import { DebugElement } from '@angular/core';
 import { By } from '@angular/platform-browser';
 import { mockDotLanguage } from '../../../test/dot-language.mock';
 import { Observable } from 'rxjs/Observable';
+import { StringPixels } from '../../../api/util/string-pixels-util';
+import { DotLanguage } from '../../../shared/models/dot-language/dot-language.model';
 
 describe('DotLanguageSelectorComponent', () => {
     let component: DotLanguageSelectorComponent;
     let fixture: ComponentFixture<DotLanguageSelectorComponent>;
     let de: DebugElement;
+    const stringPixels = new StringPixels;
```

Comment:
this should be a mock

Reply:
this is not needed anymore, so I just removed it